### PR TITLE
Update Plasma to 5.13.3

### PR DESCRIPTION
**Pull request by Johnnynator**
Cross build still need to be fixed. (Also some licenses don't follow spdx)

### CODE REVIEW

**Comment by Johnnynator** [comment]
I fixed some cross builds and set the remaining stuff to nocross. So it should be fine now.

**Comment by Johnnynator** [comment]
The Lockscreen issue was just broken mesa on musl.

File: srcpkgs/breeze-gtk/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # Template file for 'breeze-gtk'
 pkgname=breeze-gtk
-version=5.12.0
+version=5.13.3
 revision=1
 build_style=cmake
 configure_args="-DWITH_GTK3_VERSION=3.20"
```
Comment (maxice8) [inline srcpkgs/breeze-gtk/template]: isn't our version 3.22 or it doesn't matter really ?

**Comment by Johnnynator** [comment]
To be honest idk what exactly this var effects. Would have to check. But I didn't notice any uncommon behavior on my PC.

**Comment by maxice8** [comment]
it doesn't even appear on the source of the package

**Pull request merged**